.vscode.example/settings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "[javascript]": {
+    "editor.formatOnSave": true,
+    "editor.defaultFormatter": "dbaeumer.vscode-eslint"
```

Comment:
One question/quibble - we'd agreed on prettier as being a default acceptable format, and I had been using that on update/auto-fix - I want to make sure as a team we're not changing this without discussion.  

I'm fine merging the PR as I doubt this gets pulled in by anyone automatically, but we should try to avoid making this change unless we do this wholesale.   Arch meeting discussion seems fine.

Reply:
Let's talk about this because, right now, the eslint rules and the prettier rules are in conflict. I ran into this with the way that switch statements were being formatted.